index.html

```diff
@@ -151,1145 +151,1103 @@ <h4>Other Related Resources</h4>
+              "Naskh script">
 
+              <figcaption>
+                Naskh example [<a href=
+                "https://commons.wikimedia.org/wiki/File:FirstSurahKoran_%28fragment%29.jpg">Source</a>].
+              </figcaption>
+            </figure>
 
-      <p>When the main script is Arabic, the layout and structure of pages and documents are also
-      set from right to left.</p>
 
+            <p>One of the clearest styles of all, with clearly distinguished letters which
+            facilitate reading and pronunciation. Can be written at small sizes (traditionally
+            using pens made of reeds and ink), which suits the production of longer texts used for
+            boards and books intended for the general population, especially the Qur‘an.</p>
+          </dd>
 
-      <p><dfn data-lt="bidirectional algorithm|bidi algorithm"><a href=
-      "http://www.unicode.org/reports/tr9/">Unicode Bidirectional Algorithm</a></dfn> (or
-      <span class="qterm">bidi algorithm</span>, for short) [[!BIDI]] details an algorithm for
-      rendering right-to-left text and covers a myriad of situations in mixing different kinds of
-      characters. A simpler explanation of the basics of the algorithm exists in the W3C article
-      <a href="https://www.w3.org/International/articles/inline-bidi-markup/uba-basics">Unicode
-      Bidirectional Algorithm basics</a>. [[UBA-BASICS]] You can refer to these documents for more
-      information about Unicode’s bidirectional algorithm.</p>
 
+          <dt>Ruqʻa (رقعة‎)</dt>
 
-      <p>A brief overview of the <a>bidirectional algorithm</a> follows, because the direction is
-      an essential part of how Arabic script is used.</p>
 
+          <dd class="flexContainer">
+            <figure class="floatedFigure">
+              <img style="width: 129px; height: 168px;" src="images/Ruq_ah.gif" alt="Ruqʻa script">
 
-      <p>The characters of a text are digitally stored and transferred in the same order that they
-      are typed by a user. This is the order in which the text is read and pronounced by people and
-      held in memory by software applications, as shown in <a href="#figure-order_in_memory"></a>
-      for a sample text.</p>
+              <figcaption>
+                Ruqʻa example [<a href=
+                "https://fa.wikipedia.org/wiki/%D9%BE%D8%B1%D9%88%D9%86%D8%AF%D9%87:Ruq_ah.gif">Source</a>].
+              </figcaption>
+            </figure>
 
 
-      <figure id="figure-order_in_memory">
-        <img src="images/order-in-memory.svg" alt="The order of characters in memory">
+            <p>A handwritten style still commonly used in Arabic countries, and recognisable by its
+            bold-like letters written above the writing line. Designed to be used for education,
+            for everyday writing and adopted in the offices (<em>Diwan</em>) of the Ottoman Empire.
+            One of it's feature is that calligraphers have kept it and did not derived variations
```

Comment:
This sentence and grammar needs some attention.

Reply:
Yes, this is not my text but came up as new because of reorganisation.